ProcessList.c

```diff
@@ -105,13 +105,19 @@ static const char* alignedProcessFieldTitle(const ProcessList* this, ProcessFiel
    if (!title)
       return "- ";
 
-   if (!Process_fields[field].pidColumn)
-      return title;
+   if (Process_fields[field].pidColumn) {
+      static char titleBuffer[PROCESS_MAX_PID_DIGITS + /* space */ 1 + /* null-terminator */ + 1];
```

Comment:
There are two plus signs in this line. I wonder why the compiler didn't warn or fail to build this.
Can we correct it as well? I know it's present in the old code.
```
static char titleBuffer[PROCESS_MAX_PID_DIGITS + /* space */ 1 + /* null-terminator */ + 1];
                                                               ^ ~~~~~~~~~~~~~~~~~~~~~~~
```

```
static char titleBuffer[PROCESS_MAX_PID_DIGITS + /* space & null terminator */ sizeof(" ")];
```

Reply:
> There are two plus signs in this line. I wonder why the compiler didn't warn or fail to build this.

Because providing an extra sign for a number like `1 + -5` or `1 - -5` is totally legit. Just writing `1 ++5` isn't, because that could (and is) misinterpreted as a pre-increment operator (but ` 1 + +5` is valid).

> Can we correct it as well? I know it's present in the old code.

Sure. A PR may always clean up nearby code, as long as that cleanup is performed in a separate patch (if you didn't need to touch that line in the first place anyways).

> 
> ```
> static char titleBuffer[PROCESS_MAX_PID_DIGITS + /* space */ 1 + /* null-terminator */ + 1];
>                                                                ^ ~~~~~~~~~~~~~~~~~~~~~ ^
> ```
> 
> ```
> static char titleBuffer[PROCESS_MAX_PID_DIGITS + /* space & null terminator */ sizeof(" ")];
> ```

I'm not sure I like that variant …

Reply:
So the second "+" represents positive sign. I get it now, but I would expect a warning anyway unless the plus sign is expanded from a macro or the number with the sign is parenthesized.

Reply:
Why not remove the inline comment entirely, and use the `sizeof` @Explorer09 suggested?
```
static char titleBuffer[PROCESS_MAX_PID_DIGITS + sizeof(" ")];
```

This already contains the information about the space and the null terminator, and I think is actually clearer than the version with comments.

Reply:
I added a commit with this change